version.go

```diff
@@ -2,5 +2,5 @@ package awstpr
 
 const (
 	// VersionV1 is the Kubernetes API version for v1 cluster resources.
-	VersionV1 = "aws.cluster.giantswarm.io/v1"
+	VersionV1 = "cluster.giantswarm.io/v1"
```

Comment:
Its the k8s apiVersion as in this example TPO. Its also the value we were using previously.

```
kubectl get aws test -o yaml
apiVersion: cluster.giantswarm.io/v1
kind: Aws
metadata:
  creationTimestamp: 2017-05-05T12:40:34Z
  name: test
  namespace: giantswarm
  resourceVersion: "14195"
  selfLink: /apis/cluster.giantswarm.io/v1/namespaces/giantswarm/awses/test
  uid: 08b588d4-3190-11e7-a77f-aa30e12dc099
```